src/ci/docker/host-x86_64/x86_64-gnu-tools/Dockerfile

```diff
@@ -21,6 +21,15 @@ RUN sh /scripts/sccache.sh
 COPY scripts/cmake.sh /scripts/
 RUN /scripts/cmake.sh
 
+RUN curl -sL https://nodejs.org/dist/v14.4.0/node-v14.4.0-linux-x64.tar.xz | tar -xJ
+ENV PATH="/node-v14.4.0-linux-x64/bin:${PATH}"
+
+# Install required dependencies from browser-UI-test framework
+# For now, we need to use `--unsafe-perm=true` to go around an issue when npm tries
+# to create a new folder. For reference:
+# https://github.com/puppeteer/puppeteer/issues/375
+RUN npm install browser-ui-test -g --unsafe-perm=true
+
 COPY host-x86_64/x86_64-gnu-tools/checktools.sh /tmp/
 
 ENV RUST_CONFIGURE_ARGS \
```

Comment:
We'll want to:

* remove the check-aux-and-gui Makefile rule
* add to the SCRIPT here an explicit invocation of the src/test/rustdoc-gui test, likely after we call checktools.sh (via &&)